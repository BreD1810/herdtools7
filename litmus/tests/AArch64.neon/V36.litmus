AArch64 V36
(* Test Neon final state checking
   LD2 single structure *)
{
x = 22;
0:X0=x;
}
 P0                         ;
 MOVI V0.4S,#0              ; (* Note issues involving register default values *)
 MOVI V1.4S,#0              ;
 MOV W1,#1                  ;
 STR W1,[X0,#4]             ;
 LD2 {V0.S,V1.S}[0],[X0]    ;
forall(0:V0.S = {22,0,0,0} /\ 0:V1.S = {1,0,0,0})
