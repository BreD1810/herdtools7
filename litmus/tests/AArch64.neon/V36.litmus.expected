Test V36 Required
Histogram (1 states)
1000000:>0:V0.S={22,0,0,0}; 0:V1.S={1,0,0,0};
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (0:V0.S={22,0,0,0} /\ 0:V1.S={1,0,0,0}) is validated
Hash=f9ba348dcd400f89f922e87d52ac6382
Observation V36 Always 1000000 0
